config/profile/rhdh/default-config/deployment.yaml

```diff
@@ -28,16 +28,6 @@ spec:
                   requests:
                     storage: 2Gi
           name: dynamic-plugins-root
-        - name: dynamic-plugins-npmrc
-          secret:
-            defaultMode: 420
-            optional: true
-            secretName: dynamic-plugins-npmrc
-        - name: dynamic-plugins-registry-auth
-          secret:
-            defaultMode: 416
-            optional: true
-            secretName: dynamic-plugins-registry-auth
```

Comment:
I guess removing this will break the ability to load plugins from private OCI registries, documented here: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.6/html/installing_and_viewing_plugins_in_red_hat_developer_hub/assembly-third-party-plugins#proc-load-plugin-oci-image_assembly-install-third-party-plugins-rhdh

> For an Operator-based deployment, replace `<secret_name>` with `dynamic-plugins-registry-auth`.

Reply:
It does not break the ability
The way it is documented/implemented considers single dynamic-plugins-registry-auth secret per namespace, which (according to our concept) is not correct. 
It can be any secret (generated from default-config or user created and referenced in CR) of auth.json format mounted to ~/.config}/containers/auth.json

Reply:
> single dynamic-plugins-registry-auth secret per namespace, which (according to our concept) is not correct.

I thought we were recommending against multiple RHDH instances in a single namespace anyway, so in a sense, I guess it should be okay (as long as this is an optional secret), no?

> It can be any secret (generated from default-config or user created and referenced in CR) of auth.json format mounted to ~/.config}/containers/auth.json

Anyway, in such cases, I'd suggest perhaps deprecating this first and filing a docs issue rather than removing this right away (which is what you did in https://github.com/redhat-developer/rhdh-operator/pull/1297/commits/71899c135199a748842be3babd70b33d99db881a).